crates/host/src/wasmbus/handler.rs

```diff
@@ -1127,3 +1199,37 @@ impl InvocationErrorIntrospect for Handler {
         InvocationErrorKind::Trap
     }
 }
+
+// TODO(joonas): Make this more generalized so we can support non-wasmcloud-specific
+// selectors as well.
+//
+// environment variable -> WASMCLOUD_LABEL_wasmcloud__ns=my-namespace-goes-here
+// becomes:
+// SPIRE Selector -> wasmcloud:ns:my-namespace-goes-here
+async fn parse_selectors_from_host_labels(
+    host_labels: tokio::sync::RwLockReadGuard<'_, BTreeMap<String, String>>,
```

Comment:
Nit: I think this can just be `&BTreeMap` because `RwLockReadGuard` implements `Deref` for its inner type